andyb95/propertyManagement
# ability to list and view available properties

## Background ##
This PR gives landlords the ability to list their properties with all necessary info.

## Testing ##
1. Go to the "List Property" page using the nav menu
2. enter in dummy info and upload a photo. click submit.
3.  Go to "Search" page to see all properties

## To do ##
1. Additional styling for property component
2. Disable buttons while loading and add spinner circle
3. Add "View" and "Apply" buttons to property component
